.github/workflows/run_gtest.yml

```
@@ -0,0 +1,434 @@
+          
+          # Create CMakeLists.txt if it doesn't exist
+          if [ ! -f "CMakeLists.txt" ]; then
+            echo "Creating generic CMakeLists.txt..."
+            cat > CMakeLists.txt << 'EOF'
+          cmake_minimum_required(VERSION 3.10)
+          project(FastestAI_Test)
+          
+          # Set C++ standard
+          set(CMAKE_CXX_STANDARD ${{ needs.analyze-project.outputs.cpp-standard == 'c++11' && '11' || needs.analyze-project.outputs.cpp-standard == 'c++14' && '14' || needs.analyze-project.outputs.cpp-standard == 'c++17' && '17' || needs.analyze-project.outputs.cpp-standard == 'c++20' && '20' || '14' }})
+          set(CMAKE_CXX_STANDARD_REQUIRED ON)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)